app/views/chess_boards/show.html.erb

```diff
@@ -12,37 +12,72 @@
   </head>
   <body>
     <br><br><br>
-
+    
     <div class="chessboard">
 
         <% (0..7).each do |row| %>
           <div id="row">
-         <% (0..7).each do |column| %>
+        <% (0..7).each do |column| %>
            <% if (row % 2 == 0) %>
              <% if (column % 2 == 0) %>
-                <div class="white"></div>
-             <% else %>
-                <div class="black"></div>
-             <% end %>
-          <% else %>
-             <% if (column % 2 == 1) %>
-                <div class="white"></div>
-             <% else %>
-                <div class="black"></div>
-             <% end %>
-           <% end %>
-         <% end %>
-          </div>
-        <% end %>
 
-    </div>
+                <div class="white">
+                <% @pieces.each do |piece| %>
+                  <% if piece.x_position == column && piece.y_position == row %>
+                    <div class="piece" display-piece-id="<%piece.id%>">
```

Comment:
Do I write a test in piece/model/spec to check for the pieces showing up in unicode? Is that that only test that goes with this since we already have a controller test covering the game#show action.